core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -257,34 +247,10 @@ public void onFinish() {
   private TableDrawerAdapter tableDrawerAdapter;
   private RecyclerView tableDrawerRight;
   protected MainMenu mainMenu;
-  private ItemTouchHelper.Callback tabCallback = new ItemTouchHelper.Callback() {
-    @Override
-    public int getMovementFlags(@NonNull RecyclerView recyclerView,
-      @NonNull RecyclerView.ViewHolder viewHolder) {
-      return makeMovementFlags(0, ItemTouchHelper.UP | ItemTouchHelper.DOWN);
-    }
-
-    @Override
-    public void onChildDraw(@NonNull Canvas c, @NonNull RecyclerView recyclerView,
-      @NonNull RecyclerView.ViewHolder viewHolder, float dX, float dY, int actionState,
-      boolean isCurrentlyActive) {
-      super.onChildDraw(c, recyclerView, viewHolder, dX, dY, actionState, isCurrentlyActive);
-      viewHolder.itemView.setAlpha(1 - Math.abs(dY) / viewHolder.itemView.getMeasuredHeight());
-    }
-
-    @Override
-    public boolean onMove(@NonNull RecyclerView recyclerView,
-      @NonNull RecyclerView.ViewHolder viewHolder, @NonNull RecyclerView.ViewHolder target) {
-      return false;
-    }
-
-    @Override
-    public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction) {
-      closeTab(viewHolder.getAdapterPosition());
-    }
-  };
+  private ItemTouchHelper.Callback tabCallback;
   private Disposable bookmarkingDisposable;
   private boolean isBookmarked;
+  private AttributeSet attrs;
```

Comment:
why was attrs made into a field?

Reply:
Made it a field before I noticed the `isAdded()` function existed. Should revert back.